tests/data/with_methods.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+class tests.modules.withmethods.withmethods.Point {
+  PI: float {static}
+  origin: Point {static}
+  coordinates: Coordinates
+  day_unit: TimeUnit
+  hour_unit: TimeUnit
+  time_resolution: Tuple[str, TimeUnit]
+  x: int
+  y: str
+  {static} from_values(x: int, y: str, u: float, z: List[int]) -> Point
+  get_coordinates() -> Tuple(float, float)
```

Comment:
the proper syntax for the return type would be `Tuple[float, float]` (with square brackets instead of curly ones)